teamscale-maven-plugin/build.gradle.kts

```diff
@@ -38,6 +41,7 @@ tasks.register<MavenExec>("build") {
 	group = "build"
 	description = "Builds the project and runs tests"
 	dependsOn(":agent:publishToMavenLocal")
+	dependsOn(":teamscale-client:publishToMavenLocal")
```

Comment:
I recently fixed this on master in one central place. You should be able to remove this after merging master.

Reply:
just this one line or also the one above where we depend on publishing the agent?

Reply:
Yes both lines. On master we have 
```
tasks.withType<MavenExec> {
	if (name != "clean") {
		dependsOn(":agent:publishToMavenLocal")
		dependsOn(":teamscale-client:publishToMavenLocal")
		dependsOn(":impacted-test-engine:publishToMavenLocal")
	}
}
```